game-runner/src/main/java/za/co/entelect/challenge/engine/runner/GameEngineRunner.java

```diff
@@ -185,7 +185,7 @@ private void publishGameComplete(boolean matchSuccessful) throws Exception {
 
             int score = player.getGamePlayer().getScore();
 
-            if (player.getName().equals("A")) {
+            if (player.getName().substring(0, 1).equals("A")) {
```

Comment:
`player.getName()` is:-> "A - nickName" or "B - nickName"
This magic string checking only worked for the second player, because "B - whatever" is NOT equal to "A"